rspace-bench/src/test/scala/coop/rchain/lmdb/LMDBOpsBench.scala

```diff
@@ -0,0 +1,301 @@
+package coop.rchain.lmdb
+
+import java.nio.ByteBuffer
+import java.nio.file.{Files, Path}
+import java.util.concurrent.TimeUnit
+
+import coop.rchain.lmdb.LMDBOpsBench._
+import coop.rchain.rspace.Context
+import coop.rchain.shared.PathOps._
+import org.lmdbjava.DbiFlags.MDB_CREATE
+import org.lmdbjava._
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+
+import scala.util.Random
+
+class LMDBOpsBench {
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime))
+  @OutputTimeUnit(TimeUnit.MILLISECONDS)
+  @Warmup(iterations = 10)
+  @Fork(value = 1)
+  @Measurement(iterations = 10)
+  def smallBlobRoundtrip(bh: Blackhole, state: SmallLMDBOpsBenchState) = {
+    val ress = state.roundtrip(state.small, state.dbGnats, state.env)
+    assert(ress == state.small)
+    bh.consume(ress)
+  }
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime))
+  @OutputTimeUnit(TimeUnit.MILLISECONDS)
+  @Warmup(iterations = 10)
+  @Fork(value = 1)
+  @Measurement(iterations = 10)
+  def smallBlobPut(bh: Blackhole, state: SmallLMDBOpsBenchState) =
+    state.small.foreach(x => {
+      val txn = state.env.txnWrite()
+      state.dbGnats.put(txn, x._1, x._2)
+      txn.commit()
+    })
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.AverageTime))
+  @OutputTimeUnit(TimeUnit.MILLISECONDS)
+  @Warmup(iterations = 10)
+  @Fork(value = 1)
+  @Measurement(iterations = 10)
+  def smallBlobGet(bh: Blackhole, state: SmallPrefilledLMDBOpsBenchState) = {
+    val res = state.small.map(x => {
+      val txn = state.env.txnRead()
+      val r   = (x._1, state.dbGnats.get(txn, x._1))
+      txn.commit()
```

Comment:
Why do we commit a read transaction?

Reply:
it's to replicate the behaviour that is in rspace